module/documents/actor.mjs

```diff
@@ -165,12 +246,16 @@ export default class ActorEd extends Actor {
       return;
     }
     // Check if this uses karma or strain at all
-    this.takeDamage( roll.edRollOptions.strain, "standard" );
+    this.takeDamage( roll.options.strain.total, "standard", undefined, true );
```

Comment:
Accessing fields of classes from outside the class should be strictly controlled. So maybe we should add a Getter to the roll calls for that, like `get totalStrain()`